.circleci/config.yml

```diff
@@ -108,11 +108,13 @@ workflows:
             branches:
               only: master
             tags:
-              only: /^v[0-9]+(\.[0-9]+)*$/
+              only: /^v.*/
       - microsite:
           requires:
             - release
           filters:
+            branches:
+              ignore: /.*/
             tags:
-              only: /^v[0-9]+(\.[0-9]+)*$/
+              only: /^v.*/
```

Comment:
Not sure about this one. The other changes are good to go imho.

Reply:
You probably want this with a `v` in front: https://github.com/semver/semver/pull/460/files#diff-55782f4857216898d46c6eca25fc5f32R334 :D

Reply:
Done @NeQuissimus can you re-review? Ty sir